geo-types/src/multi_point.rs

```diff
@@ -1,11 +1,11 @@
-use crate::{CoordNum, Point};
-
+use crate::{CoordNum, NoValue, Point};
 #[cfg(any(feature = "approx", test))]
 use approx::{AbsDiffEq, RelativeEq};
-
 use std::iter::FromIterator;
 
-/// A collection of [`Point`s](struct.Point.html). Can
+/// A generic line string with 3D space + Measure value support.
```

Comment:
copy/paste errro - this isn't a line string 

(and same as I mentioned elsewhere, I dont think  the addition of the term "generic" is clear in the context of these docs)

Reply:
fixed